src/main/java/seedu/pill/util/TransactionManager.java

```diff
@@ -98,6 +98,8 @@ public Transaction createTransaction(String itemName, int quantity,
     public Order createOrder(Order.OrderType type, ItemMap itemsToOrder, String notes) {
         Order order = new Order(type, itemsToOrder, notes);
         orders.add(order);
+        System.out.println("Order placed! Listing order details");
```

Comment:
I like how you included a user-friendly message 'Order placed! Listing order details' here